src/common/webview.cpp

```diff
@@ -224,6 +225,51 @@ wxString wxWebView::GetUserAgent() const
     return userAgent;
 }
 
+bool wxWebView::RunScript(const wxString& javascript, wxString* output) const
+{
+    m_syncScriptResult = -1;
+    m_syncScriptOutput.Clear();
+    RunScriptAsync(javascript, (void*)this);
+
+    // Wait for script exection
+    while (m_syncScriptResult == -1)
+        wxYield();
+
+    if (m_syncScriptResult && output)
+        *output = m_syncScriptOutput;
+    return m_syncScriptResult == 1;
+}
+
+void wxWebView::RunScriptAsync(const wxString& WXUNUSED(javascript),
+    void* WXUNUSED(clientData)) const
+{
+    wxLogError(_("RunScriptAsync not supported"));
+}
+
+void wxWebView::SendScriptResult(void* clientData, bool success,
+    const wxString& output) const
+{
+    // If currently running sync RunScript() don't send an event, but use
```

Comment:
_Very_ minor, but I had some trouble parsing this comment without the comma:
```suggestion
    // If currently running sync RunScript(), don't send an event, but use
```